### ISSUE TITLE
Expose `SCAN`

### ISSUE THREAD

**Issue by rubenstolk** [dial-once/node-cache-manager-redis#71]
As per Redis documentation, we shouldn't rely on `KEYS` as much but rather use `SCAN`.

**Comment by jeff-kilbride** [comment]
This was solved with #72.

**Comment by mrister** [comment]
Yup, @jeff-kilbride solved that like a pro ;-). We need to do a release to expose the changes, so I will keep open this issue until then.
